README.md

```diff
@@ -2,9 +2,90 @@
+| Guitar | String instrument     |    Music |
+| Violin | String instrument      |    Music |
+
+Let's assume you have your two datasets in a project. Now you can : 
+- Navigate to the Flow
+- Click on the + RECIPE button and access the Text Analysis menu. If your dataset is selected, you can directly find the plugin on the right panel.
+
+#### Parameter settings
+
+##### Input parameters
+
+
+
+*   Text dataset:
+    *   Text column
+    *   Language: the language parameter lets you choose among 59 supported languages if your text is monolingual. Else, the Multilingual option will let you set a Language column.This Language column parameter should contain ISO 639-1 supported language codes. 
+*   Ontology dataset:
+    *   tag column
+    *   keyword column
+    *   (Optional) category column
+
+ 
+##### Matching parameters
+
+You can widen the search by activating the following matching parameters:
+
+*   Button normalize case: To match documents by ignoring case (e.g will try to match 'guitar','Guitar', 'GUITAR')
+*   Button normalize diacritics: To match documents by ignoring diacritics marks like accents, cedillas, tildes
+*   Button lemmatization: To match documents by simplifying words to their lemma form (e.g. going → go, mice → mouse). This option supports 28 languages. 
+###### List of the 28 supported languages for lemmatization in ISO 639-1 format:
+de, es, fr, nb, pl, ru, ca, cs, da, en, hr ,hu , id, it, lb, lt, mk, nb, nl, pt, ro, sr, sv, tl, tr, ur, bn, el, fa
+##### Output format
+
+There are three available output formats. All the outputs preserve the initial datas of the dataset.
+* Default output “one row per document (array columns)”: A row for each document, with additional columns:
+  *   the assigned tags of the document (array-like)
+  *   the keywords that match with the document (array-like)
+  *   the concatenated sentences that matched the keywords (string-like)
+  However, if you gave a 'category column' in the matching parameters, you won't have a unique tag column but one column per category, each one containing a list of associated tags
+ 
+* Output “one row per match”: A row per document per match. This mode creates duplicates from the input dataset, as you can have multiple matches in a document. 
+In this case, the additional columns are:
+  * a tag column (string-like)
+  * a keyword column (string-like)
+  * a sentence column (string-like)
+  * (Optional) a category column (string-like)
+  
+* Output “one row per document (nested JSON)”: A row for each document, with additional columns:
+  * tag_json_full column : a dictionary with details about occurences of each tag, matched keywords for each tag, and the sentences where they appear. (Object-like)
+  * If you gave a 'category column' in the matching parameters, you will also have a column 'tag_json_categories' (Object-like), which is a simplified version of the previous dictionary: it contains categories (keys) and associated tags (values)
+
```

Comment:
Maybe give example json outputs here ?

Reply:
Yes ! I have planned to add pictures to have examples of it